src/ucp/wireup/wireup_cm.c

```diff
@@ -961,6 +1168,8 @@ ucs_status_t ucp_ep_cm_connect_server_lane(ucp_ep_h ep,
         return status;
     }
 
+    cm_wireup_ep               = ucp_ep_get_cm_wireup_ep(ep);
+    cm_wireup_ep->cms_used_idx = cms_used_idx;
```

Comment:
the same, better to store cm object

Reply:
saving the index is simpler since need to go over the priority list with it.

Reply:
how they are related? this is index to CM obj on worker

Reply:
not only. these are the indexes of the priority list as well - 
https://github.com/openucx/ucx/blob/master/src/ucp/core/ucp_context.h#L236